src/Blazor.Gitter.Core/Components/Pages/Room.razor.cs

```diff
@@ -19,7 +20,26 @@ public class RoomModel : ComponentBase
         string LastRoom = string.Empty;
         internal bool KeepWatching = true;
 
-        protected bool IsLoading;
+        protected bool IsLoading = true ;
```

Comment:
Can you please do something nice for the loader here - I tried and failed by copying the Search loader...didn't work.

Reply:
I mean in the CSS of course...

Reply:
I've just run the code and the loader seems to work as expected...